src/variable.cpp

```diff
@@ -81,6 +91,9 @@ std::string Variable::doSerialisation(FORMATS format) const
         if (getInitialValue().length()) {
             repr += " initial_value=\"" + getInitialValue() + "\"";
         }
+        if (getInterfaceType() != Variable::INTERFACE_TYPES::INTERFACE_TYPE_UNDEFINED) {
```

Comment:
My read of the 1.2 Spec suggested that there is an expectation of being able to explicitly specify `interface=none` but if you and others disagree I would be happy to take that into account.